packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -11,17 +14,24 @@ const SearchSuggestions = ({
   handleSuggestionClickSubmit,
   disableSuggestionRedirect,
 }) => {
-  const getLinkToCourse = (course) => `/${enterpriseSlug}/course/${course.key}`;
+  const getLinkToCourse = (course) => {
+    const { learning_type: learningType } = course;
+    if (learningType === LEARNING_TYPE_EXECUTIVE_EDUCATION) {
```

Comment:
nit/suggestion: Perhaps make this open to future external course types as well? For example, might consider doing something like:

```jsx
const externalCourseLearningTypes = ['LEARNING_TYPE_EXECUTIVE_EDUCATION'];
if (externalCourseLearningTypes.includes(learningType) {
  return `/${enterpriseSlug}/${learningType}/course/${course.key}`;
}
```

Reply:
@adamstankiewicz The URL does not contain learning type (`Executive Education`) it contains course type (i.e. `executive-education-2u`). So we need to handle each learning type separately, or we will need to create a mapping object to map `learningType` to `courseType`. 

Should I create the mapping now? or wait till we have more than 1 learning types?